framework/include/base/MooseError.h

```diff
@@ -140,4 +140,23 @@
 
 #define mooseCheckMPIErr(err) do { if (err != MPI_SUCCESS) { if (libMesh::global_n_processors() == 1) print_trace(); libmesh_here(); MPI_Abort(libMesh::GLOBAL_COMM_WORLD,1); exit(1); } } while (0)
 
+#define mooseInfo(msg)                                                  \
+  do                                                                    \
+  {                                                                     \
+    mooseDoOnce(                                                        \
+      {                                                                 \
+        std::ostringstream _info_oss_;                                  \
+                                                                        \
+        _info_oss_                                                      \
+          << (Moose::_color_console ? XTERM_BLUE : "")                  \
```

Comment:
I'm not completely against using blue but it probably doesn't show up well on black backgrounds.